lib/internal/policy/manifest.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+const {
+  ERR_MANIFEST_ASSERT_INTEGRITY,
+  ERR_MANIFEST_INTEGRITY_MISMATCH,
+  ERR_MANIFEST_UNKNOWN_ONERROR,
+} = require('internal/errors').codes;
+const SRI = require('internal/policy/sri');
+const { SafeWeakMap } = require('internal/safe_globals');
+const crypto = require('crypto');
+const { Buffer } = require('buffer');
+const { URL } = require('url');
+const { createHash, timingSafeEqual } = crypto;
+const Hash$update = Function.call.bind(crypto.Hash.prototype.update);
+const Hash$digest = Function.call.bind(crypto.Hash.prototype.digest);
+const Buffer$equals = Function.call.bind(Buffer.prototype.equals);
+const Buffer$toString = Function.call.bind(Buffer.prototype.toString);
+const RegExp$test = Function.call.bind(RegExp.prototype.test);
+const entries = Object.entries;
+const INTEGRITIES = new SafeWeakMap();
+const REACTIONS = new SafeWeakMap();
+const RELATIVE_URL_STRING_PATTERN = /^\.{0,2}\//;
+const { shouldAbortOnUncaughtException } = process.binding('config');
+const { abort, exit, _rawDebug } = process;
+function REACTION_THROW(error) {
+  throw error;
+}
+function REACTION_EXIT(error) {
+  REACTION_LOG(error);
+  if (shouldAbortOnUncaughtException) {
+    abort();
+  }
+  exit(1);
+}
+function REACTION_LOG(error) {
+  _rawDebug(error.stack);
+}
+class Manifest {
+  constructor(obj, manifestURL) {
+    const integrities = {
+      __proto__: null,
+    };
+    const reactions = {
+      __proto__: null,
+      integrity: REACTION_THROW,
```

Comment:
Why is this property not just named `onerror`, or something like `callback`?

Reply:
eventually there could be many types of reactions for things like attempting to load dependencies that are not allowed. This code is a stripped down version of manifests that do such enforcement.